library/src/main/java/com/firebase/ui/auth/core/FirebaseLoginDialog.java

```diff
@@ -52,17 +53,17 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (mTwitterAuthHelper != null) {
```

Comment:
The app user clicked only one button to sign in. We should only call `onActivityResult()` for that provider.